libraries/docker_service_manager_systemd.rb

```diff
@@ -26,6 +26,16 @@ class DockerServiceManagerSystemd < DockerServiceBase
         not_if { docker_name == 'default' && ::File.exist?('/lib/systemd/system/docker.socket') }
       end
 
+      template '/etc/systemd/system/containerd.service' do
```

Comment:
I don't think containerd goes all the way back to 17.12. You might want to figure out when it came out and then mark that as the oldest version of Docker supported.

Reply:
That thought had occurred to me. I'll look into that.